src/libraries/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.xml

```diff
@@ -156,6 +156,15 @@
             <param name="elementOffset">The offset to add.</param>
             <returns>A new reference that reflects the addition of offset to pointer.</returns>
         </member>
+        <member name="M:System.Runtime.CompilerServices.Unsafe.Add``1(``0@,System.UIntPtr)">
```

Comment:
This is still referencing UIntPtr

Reply:
I think it has to, doesn't it? What else would this reference since `nuint` isn't a "real" type?

Reply:
Hmm, you are right. It feels important to be able to capture differences between UIntPtr and nuint in the docs.

@carlossanlop Do you have suggestion how this should be handled?

Reply:
I've verified that compiler generated xml doc uses `UIntPtr` to represent `nuint`.